src/asm/aarch64/predict.rs

```diff
@@ -68,12 +68,21 @@ decl_cfl_pred_fn! {
 pub fn dispatch_predict_intra<T: Pixel>(
   mode: PredictionMode, variant: PredictionVariant,
   dst: &mut PlaneRegionMut<'_, T>, tx_size: TxSize, bit_depth: usize,
-  ac: &[i16], angle: isize, edge_buf: &AlignedArray<[T; 4 * MAX_TX_SIZE + 1]>,
-  cpu: CpuFeatureLevel,
+  ac: &[i16], angle: isize, angle_delta: i8,
```

Comment:
Maybe those 3 arguments could be wrapped in a struct or enum?

Reply:
struct of delta_y and delta_uv is added.
angle is not added to the struct because its corresponding to PredictionMode
which exist in PartitionParameters.